clusterloader2/pkg/measurement/interface.go

```diff
@@ -28,6 +29,8 @@ type MeasurementConfig struct {
 	// into the Execute method.
 	Params map[string]interface{}
 	// TODO(krzysied): add CloudProvider.
```

Comment:
Why do you need CloudProvider?

Reply:
TBH, it was suggested by @shyamjvs . At this moment I'm not sure whether we will need cloud provider or not, so I would like to keep the TODO.